### DISCUSSION TITLE
Level 4 Testing attribute requirements

### DISCUSSION THREAD

**Discussion by BarboneAllplan** [jwouellette/TPF-5_372-Unit_Test_Suite#7] [General]
For the Level 4 testing, are there specific attributes that the vendors are requested to consider or use? Any attributes required to identify elements within the IFC4.3 schema will be included, but are there any other attributes that should be considered (e.g., material, pay item codes, reinforcement mark code, etc.)?

**Comment by jwouellette** [comment]
We're working on that. The requirements will be a combination of Excel workbooks and PDFs. 
We will include classification via the respective state Pay Item codes
